### PR TITLE
Magecloud 5091: Update devdocs documentation with changes in MAGECLOUD-2880

### PR DESCRIPTION
**Pull request by oshmyheliuk**
## Purpose of this pull request

This pull request will update the information about changes in db-dump command.

## Affected DevDocs pages

-  https://devdocs.magento.com/cloud/project/project-webint-snap.html#db-dump

## Magento source code changes

https://github.com/magento/ece-tools/pull/662

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: src/cloud/project/project-webint-snap.md [hunk 1]
```diff
@@ -163,9 +163,10 @@ To create a database dump:
 
  {:.bs-callout-info}
 
--  We recommend putting the application in maintenance mode before doing a database dump in Production environments.
+-  Before creating dump the command will put the application in maintenance mode. Also, command stops all crons and kill all running consumers and crons.
```
Comment (meker12) [inline src/cloud/project/project-webint-snap.md]:
```suggestion
-  Before creating a dump, the command switches the application to maintenance mode and stops all crons, which terminates all running consumers and cron jobs.   
```

File: src/cloud/project/project-webint-snap.md [hunk 2]
```diff
@@ -163,9 +163,10 @@ To create a database dump:
 
  {:.bs-callout-info}
 
--  We recommend putting the application in maintenance mode before doing a database dump in Production environments.
+-  Before creating dump the command will put the application in maintenance mode. Also, command stops all crons and kill all running consumers and crons.   
 -  This command creates an archive file for each database backup with the file name pattern `dump-<label><timestamp>.sql.gz` where  _label_ is replaced with the database name.  The archive files are saved to your remote project directory, and the path to each file is listed in the command output.
 -  If an error occurs during the dump, the command deletes any dump files to conserve disk space. Review the logs for details (`var/log/cloud.log`).
+-  In the end the command will remove the flag about maintenance mode and will re-enable cron processes.
```
Comment (meker12) [inline src/cloud/project/project-webint-snap.md]:
```suggestion
-  After the dump operation completes, the command switches the application back to production mode and re-enables cron processes.
```

File: src/cloud/project/project-webint-snap.md [hunk 3]
```diff
@@ -163,9 +163,10 @@ To create a database dump:
 
  {:.bs-callout-info}
 
--  We recommend putting the application in maintenance mode before doing a database dump in Production environments.
+-  Before creating dump the command will put the application in maintenance mode. Also, command stops all crons and kill all running consumers and crons.   
 -  This command creates an archive file for each database backup with the file name pattern `dump-<label><timestamp>.sql.gz` where  _label_ is replaced with the database name.  The archive files are saved to your remote project directory, and the path to each file is listed in the command output.
 -  If an error occurs during the dump, the command deletes any dump files to conserve disk space. Review the logs for details (`var/log/cloud.log`).
```
Comment (meker12) [inline src/cloud/project/project-webint-snap.md]: If the dump fails, do you have to do something about switching back to production mode and restarting cron processes?

Comment (oshmyheliuk) [reply src/cloud/project/project-webint-snap.md]: Yes, in case of fail command will switch back application to production and start crons.

**Pull request merged**